src/main/java/graphql/schema/Coercing.java

```diff
@@ -59,8 +90,30 @@
      *
      * @throws graphql.schema.CoercingParseValueException if value input can't be parsed
      */
+    @Deprecated
     @NotNull I parseValue(@NotNull Object input) throws CoercingParseValueException;
 
+    /**
+     * Called to resolve an input from a query variable into a Java object acceptable for the scalar type.
+     * <p>
+     * Note : You should not allow {@link java.lang.RuntimeException}s to come out of your parseValue method, but rather
+     * catch them and fire them as {@link graphql.schema.CoercingParseValueException} instead as per the method contract.
+     *
+     * @param input          is never null
+     * @param graphQLContext the graphql context in place
+     * @param locale         the locale to use
+     *
+     * @return a parsed value which is never null
```

Comment:
Since this method doesn't carry the `@NotNull` return annotation anymore, should this comment be updated?

Reply:
Thanks, that's a good catch. I'll fix it up